apps/web/vibes/soul/sections/product-description/index.tsx

```diff
@@ -8,41 +9,83 @@ export interface AccordionItem {
 }
 
 interface Props {
-  accordions: AccordionItem[];
-  image?: {
+  accordions: Streamable<AccordionItem[] | null>;
```

Comment:
Actually, do arrays still need `null`? 🤔

Reply:
Yeah, I think we should do it to be consistent. That being said, the `accordions` prop is required so for that reason we shouldn't have `null` here